### PR TITLE
TA Status Cards

### PR DESCRIPTION
**Pull request by dajinchu**
image

### CODE REVIEW

File: packages/app/components/RenderEvery.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect, useCallback, ReactElement } from "react";
+
+interface RenderEveryProps {
+  render: () => ReactElement;
+  interval: number;
+}
+export function RenderEvery({
```
Comment (willstenzel) [inline packages/app/components/RenderEvery.tsx:7]: This is slick, I like it

File: packages/app/components/Queue/StudentQueueList.tsx [hunk 2]
```diff
@@ -299,6 +300,7 @@ export default function StudentQueueList({
                 )}
               </Col>
             </Row>
+            <StatusRow questions={questions} taList={queue.staffList} />
```
Comment (willstenzel) [inline packages/app/components/Queue/StudentQueueList.tsx:303]: This kinda removes the anonymity added in https://github.com/sandboxnu/office-hours/pull/191

I'm not sure what makes the most sense regarding student anonymity, might be worth having a discussion about.

File: packages/app/components/Queue/StatusRow.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  helpedAt: Date;
+}
+function HelpingFor({ studentName, helpedAt }: HelpingForProps): ReactElement {
+  return (
+    <RenderEvery
+      render={() => (
+        <span>
+          Helping <BlueSpan>{studentName}</BlueSpan> for{" "}
+          <BlueSpan>
+            {Math.round((Date.now() - helpedAt.getTime()) / 60000) + " min"}
```
Comment (liustanley) [inline packages/app/components/Queue/StatusRow.tsx:115]: We've been putting time related calculations into `/util/TimeUtil.ts` if you want to add this into there

File: packages/app/components/Queue/StudentQueueList.tsx (hunk 2 above)
Comment (liustanley) [reply packages/app/components/Queue/StudentQueueList.tsx:303]: Yeah, how about for TA queues we keep it the same, but for a student queue all they can see is "Helping a student for 23 minutes"

**Pull request merged**